### ISSUE TITLE
Implement `np.ascontiguousarray` for Numba scalars

### ISSUE THREAD

**Issue by brandonwillard** [numba/numba#7353]
The conversion for `np.ascontiguousarray` doesn't appear to handle scalar inputs:
```python
import numpy as np
import numba as nb


@nb.njit
def test_fn(x):
    return np.ascontiguousarray(x)


test_fn(1)
# ...
# TypingError: Failed in nopython mode pipeline (step: nopython frontend)
# No implementation of function Function(<function ascontiguousarray at 0x7fdd294aab00>) found for signature:
#
#  >>> ascontiguousarray(int64)


np.ascontiguousarray(1)
# array([1])
```

**Comment by JohamyB** [comment]
Hi @gmarkall ,
I'm new here and want to take this one as my first issue?

**Comment by JohamyB** [comment]
Hi,
I have been checking the documentation and the code, but I'm still not able to debug, can you please provide me a guide on this?

**Comment by JohamyB** [comment]
@stuartarchibald  I have tried with this: https://llvmlite.readthedocs.io/en/v0.25.0/admin-guide/install.html but I'm still not able to update llvmlite.  Do you know if I'm missing something else?

Installed c:\users\laptop1\anaconda3\lib\site-packages\numba-0.55.0.dev0+502.g453765375.dirty-py3.7-win-amd64.egg
Processing dependencies for numba==0.55.0.dev0+502.g453765375.dirty
Searching for llvmlite<0.39,>=0.38.0dev0
Reading https://pypi.org/simple/llvmlite/
No local packages or working download links found for llvmlite<0.39,>=0.38.0dev0
error: Could not find suitable distribution for Requirement.parse('llvmlite<0.39,>=0.38.0dev0')

**Comment by stuartarchibald** [comment]
@JohamyB the issue is that you need a "development" version of `llvmlite`. `conda` packages are the way to obtain these: https://llvmlite.readthedocs.io/en/stable/admin-guide/install.html#pre-built-binaries and as noted in the docs:
> Development releases of binary wheels are not made available.

Alternatively you can build them yourself but they are quite challenging: https://llvmlite.readthedocs.io/en/stable/admin-guide/install.html#building-manually

**Comment by DhruvPatel01** [comment]
@JohamyB Are you still working on this? I would like to work on it, if you are not.

**Comment by JohamyB** [comment]
Hi @DhruvPatel01 ,
Sure, you can work on this.

**Comment by jeertmans** [comment]
Hello ! Shouldn't this issue be closed since #7700 was merged?
